bankid/certutils.py

```diff
@@ -19,10 +19,12 @@
 
 
 def resolve_cert_path(file: str) -> pathlib.Path:
-    return importlib.resources.files("bankid.certs").joinpath(file)
+    path = importlib.resources.files("bankid.certs").joinpath(file)
+    assert isinstance(path, pathlib.Path)
```

Comment:
The return type of `joinpath` is `importlib.abc.Traversable`, hence the assertion.

Reply:
I generally avoid `assert`s in code that is run, and I don't really think it is needed? The `resolve_cert_path` is an internally used method providing no value to end user, so I don't think it is needed.

It doesn't hurt either. Your call.

Reply:
It's needed to let the mypy know (that we know) that the variable actually is an instance of `pathlib.Path`. We could solve it with an ignore or cast but I generally prefer assertions in these cases. Another option is to expose the actual `importlib.abc.Traversable` type as the return type. I'll think about it :)